packages/kbn-server-route-repository/src/register_routes.ts

```diff
@@ -122,12 +119,16 @@ export function registerRoutes<TDependencies extends Record<string, any>>({
 
     logger.debug(`Registering endpoint ${endpoint}`);
 
+    const validationObject = isZod(params)
+      ? makeZodValidationObject(params as ZodParamsObject)
+      : routeValidationObject;
```

Comment:
can we rename this to something more descriptive now that we also support zod, something like `passThroughValidationObjectForIoTs`?

Reply:
I'll rename it! 

Zod has a `.strict` modifier you can use to get the same effect as our `strictKeysRt`

Reply:
Fixed